jitpack.yml

```diff
@@ -0,0 +1,3 @@
+install:
+  - sbt -v +publishM2
```

Comment:
Could you please explain what `publishM2` does?

Reply:
So the `publishM2` command will publish to a local maven repo. Jitpack requires this in order to publish the artefacts to their hosted repo, [as per their SBT-specific instructions](https://docs.jitpack.io/building/#sbt-projects).

Reply:
Thank you